engines/director/detection_tables.h

```diff
@@ -6058,6 +6059,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME2t_l("glassyocean", "", "START.EXE",		  "c5dcc03aca7e0bed95844afdd86f866c", 1410529,
 								   "DATA/TAMURA.DXR", "a7bd15f6f42162fed7f32e0a739617ec", 155168, Common::JA_JPN, 501),
 
+	WINGAME1("gothos", "", "gothos.exe", "3fbb58682d34ce5ec8b774064725b172", 1788333, 501),
```

Comment:
Could you please provide the tail md5s instead of head ones?

Reply:
Fixed